src/core/AudioEngine.cpp

```diff
@@ -240,6 +240,15 @@ void AudioEngine::stopProcessing()
 }
 
 
+void AudioEngine::startExporting(const struct qualitySettings& qs)
```

Comment:
Shouldn't there be a finishExporting too?

Reply:
Else there's chances of lmms becoming unresponsive after export.

Reply:
Finish exporting would run `AudioEngine::stopProcessing()`. I could add in that function if you want.

Reply:
If on finish, processing is stopped, then why do you need to stop processing on starting export too?